test/integration/scripting_spec.mjs

```diff
@@ -1764,11 +1764,8 @@ describe("Interaction", () => {
       // it is usually very fast and therefore activating the selector check
       // too late will cause it to never resolve because printing is already
       // done (and the printed page div removed) before we even get to it.
-      pages = await loadAndWait(
-        "autoprint.pdf",
-        "",
-        null /* zoom = */,
-        async page => {
+      pages = await loadAndWait("autoprint.pdf", "", null /* zoom = */, {
+        postPageSetup: async page => {
```

Comment:
I assume we can't do this in the `prePageSetup` step? The reason I'm asking is because there is a small (tiny?) risk here that once we do `page.goto()` and quickly load the autoprint PDF that the print event appears on the event bus before this ran, and then we try to get `globalThis.printResolve` which is not set yet. Or is that not a concern here?